packages/utils/src/fhir-converter/integration-test.ts

```diff
@@ -189,7 +189,30 @@ function storeStats(stats: any, statsLocation: string) {
 async function createTenant() {
   const config = { cxId: tenantId, organizationNumber };
   console.log(`Creating tenant @ FHIRServer w/ ${JSON.stringify(config)}`);
-  await adminFhirApi.createTenant(config);
+  try {
+    await adminFhirApi.createTenant(config);
+    console.log(`Tenant created successfully.`);
+
+    // Verify tenant creation
+    try {
+      const verifyResponse = await fhirApi.search("Patient", "_summary=count");
+      console.log(`Tenant verification successful:`, {
+        tenantId,
+        total: verifyResponse.total,
+      });
+    } catch (verifyError) {
+      console.error(`Failed to verify tenant:`, verifyError);
+      throw verifyError;
+    }
```

Comment:
NABD, but wouldn't it be better to have this after the try/catch for tenant creation? We would log twice and indicate failed to create tenant if it fails to validate.